File: 3289-the-two-sneaky-numbers-of-digitville/3289-the-two-sneaky-numbers-of-digitville.java
class Solution {
    public int[] getSneakyNumbers(int[] nums) {
        HashMap<Integer,Integer> map= new HashMap<>();
        for(int i=0;i<nums.length;i++)
        {
            map.put(nums[i],map.getOrDefault(nums[i],0)+1);
        }
        int[] result = new int[2];
        int j=0;
        for(int n:map.keySet())
        {
            if(map.get(n)==2)
            {
                result[j++]=n;
            }
        }
        return result;
    }
}

File: 3289-the-two-sneaky-numbers-of-digitville/3289-the-two-sneaky-numbers-of-digitville.py
class Solution:
    def getSneakyNumbers(self, nums: List[int]) -> List[int]:
        map = {}
        for i in nums:
            map[i]=map.get(i,0)+1
        lst = []
        for i in map.keys():
            if map[i]==2:
                lst.append(i)
        return lst
            